### DISCUSSION TITLE
Registration

### DISCUSSION THREAD

**Discussion by LauraSessions** [IBAHCM/RPiR#59] [Archive]
Please can i be added to the organisation?

**Comment by richardreeve** [comment]
Okay, I've sent you an invitation - let me know if you have any problems.

**Comment by LauraSessions** [comment]
Everything seems fine - thanks. L

​Laura Sessions BVM&S, MVetSci
PhD Research Student

Institute of Biodiversity, Animal Health & Comparative Medicine,
Graham Kerr Building,
College of Medical, Veterinary & Life Sciences | University of Glasgow | Glasgow | G12 8QQ

E-mail: l.sessions.1@research.gla.ac.uk<mailto:Daniel.Haydon@glasgow.ac.uk>
Mobile: 07526 275534

________________________________
